test/tools/router_check/json/tool_config_schemas.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+namespace Json {
+
+class ToolSchema {
+
+public:
+  /**
+   * Obtain the router check json schema
+   * @return const std::string& of the schema string
+   */
+  static const std::string& routerCheckSchema();
```

Comment:
Originally I followed the definition pattern in config_shemas.h but I changed to wrapping the variable in a function in this commit: https://github.com/lyft/envoy/pull/682/commits/34d1dcbcd074a450284203f71738419e0e1418ae.

The reasoning was in response to comments pointing out that static const string types are dangerous due to random ordering of constructor and destructor calls.